src/components/Datasets/datasets.util.js

```diff
@@ -128,12 +129,16 @@ export const generateDataSetsDetailsMenu = selectedItem => [
   }
 ]
 
-export const generatePageData = (handleRequestOnExpand, isSelectedItem) => ({
+export const generatePageData = (handleRequestOnExpand, isSelectedItem, selectedItem) => ({
   actionsMenuHeader,
   details: {
     menu: [],
     infoHeaders,
-    type: DATASETS
+    type: DATASETS,
+    additionalInfo: {
+      header: 'Producer',
+      body: generateProducerDetailsInfo(selectedItem)
```

Comment:
add one more property - hidden, where you'll set if we need to show this additional inform, in this case, it depends on if selectedItem has producer, if so then show this info, if not - don't show anything